content/20.treatments.md

```diff
@@ -186,7 +186,81 @@ Summarize the state of the antiviral approach.
 
 ### Broad-Spectrum Pharmaceuticals
 
-#### Hydroxychloroquine
+##### Hydroxychloroquine and Chloroquine
+
+Hydroxychloroquine (HCQ) and chloroquine (CQ) are quinolones that have been widely used as anti-malarial and disease-modifying anti-rheumatic drugs in humans.
+HCQ and CQ act through the same mechanism of action, but HCQ has a better safety profile [@doi:10.1136/ard.2008.101766].
+HCQ/CQ have been reported to have anti-viral effects, including in HIV-1, SARS-CoV, MERS-CoV, and SARS-CoV-2 [@doi:10/dh8phb; @doi:10.1128/AAC.03011-14; @doi:10.1093/cid/ciaa237; @doi:10.1038/s41422-020-0282-0].
+In vitro studies of CQ use for SARs-CoV infection suggest anti-viral activity is due to inhibition of viral entry through elevation of endosomal pH and terminal glycosylation of angiotensin-converting enzyme 2, the cellular entry receptor [@doi:10.1186/1743-422X-2-69].
+In vitro studies of SARS-CoV-2 infection show both HCQ and CQ to be effective in inhibiting viral replication, with HCQ being more potent [@doi:10.1093/cid/ciaa237].
+Thus, there is excitement about HCQ as a potential therapeutic for SARS-CoV-2 and COVID-19.
+This section will discuss current efforts on assessing the potential role of HCQ in treating COVID-19.
+
+###### Current Evidence
+
+Gautret et al conducted a non-randomized, non-blinded, non-placebo clinical trial on 42 hospitalized patients comparing HCQ to standard care [@doi:10.1016/j.ijantimicag.2020.105949].
+This trial found patients who received HCQ showed higher rates of virological clearance by nasopharyngeal swab on Days 3-6 when compared to standard care.
+This study also treated six patients with both HCQ + azithromycin and found this combination therapy to be more effective than HCQ alone.
```

Comment:
Should we remove this study since it is being retracted? Or add a point that it has been retracted?

Reply:
It was posted to medRxiv, as a pre-print
https://www.medrxiv.org/content/10.1101/2020.03.16.20037135v1
Maybe we can use that as a citation?

Reply:
I'll add a point detailing the Retraction Watch post. (I don't think it's been officially retracted yet - just official comments.)

Reply:
@hufengling if you like the phrasing @rando2 suggested below (https://github.com/greenelab/covid19-review/pull/146#discussion_r404739396) you could commit those changes.  I like her phrasing there.

Reply:
I just saw and committed those!